src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/10.2-4.sql

```diff
@@ -12,15 +12,19 @@ BEGIN
 
     IF substring(current_Setting('server_version'), '\d+')::int >= 14 THEN
     EXECUTE $cmd$
+        SET citus.enable_ddl_propagation TO OFF;
```

Comment:
Why do we need to disable ddl propagation here? array_cat_agg should be an extension owned aggregate and we shouldn't propagate it.